doc/install.rst

```diff
@@ -133,6 +133,10 @@ useful::
 
 By default, workers listen to all queues.
 
+For message brokers that support prefixed-queue names, SQUAD has the optional
+setting `SQUAD_CELERY_QUEUE_NAME_PREFIX`, that, if set, will prepend it before
```

Comment:
it's actually an environment variable, not a setting. it needs to be listed also in `doc/install.rst`, together with the rest of supported environment variables

Reply:
Right! I reworded `setting` to `environment variable`, the correct name indeed. Also added it to the environment variables below in the same file.